docker.go

```diff
@@ -757,6 +758,15 @@ func sanitizeCacheCommand(build *Build) {
 				}
 			}
 
+			if build.PathStyle && strings.Contains(arg, "type=s3") {
+				if strings.Contains(arg, "use_path_style=false") {
```

Comment:
Why shouldn't the provided arg be honored here? Please add comment.

Reply:
I thought PLUGIN_PATH_STYLE should be the final override. Discussed with @jamie-harness as well and he also thinks it should not be so removed it and added a warning